adapter/service/outlvlswitch/routing.go

```diff
@@ -0,0 +1,160 @@
+package outlvlswitch
+
+import (
+	"fmt"
+
+	"github.com/futurehomeno/fimpgo"
+
+	"github.com/futurehomeno/cliffhanger/adapter"
+	"github.com/futurehomeno/cliffhanger/router"
+)
+
+// Constants defining routing service, commands and events.
+const (
+	CmdLvlSet       = "cmd.lvl.set"
+	CmdLvlStart     = "cmd.lvl.start"
+	CmdLvlStop      = "cmd.lvl.stop"
+	CmdBinarySet    = "cmd.binary.set"
+	CmdLvlGetReport = "cmd.lvl.get_report"
+
+	EvtLvlReport    = "evt.lvl.report"
+	EvtBinaryReport = "evt.binary.report"
+
+	OutLvlSwitch = "out_lvl_switch"
+	Duration     = "duration"
```

Comment:
Pure aesthetics: bundle interfaces together and service name alone.
<img width="611" alt="image" src="https://user-images.githubusercontent.com/31538120/174730153-5f290adc-feb9-4204-a106-5c5c869b5c8c.png">